azure-functions/number-generator-function/api.js

```diff
@@ -51,15 +57,26 @@ const postToAPI = async (endpoint, payload) => {
   return response;
 };
 
+/**
+ * UKEF deal ID validation check
+ * @param {String} dealId UKEF deal ID
+ * @returns {Object} Response object
+ */
 const checkDealId = (dealId) => getAPI(`deal/${dealId}`);
+
+/**
+ * UKEF facility ID validation check
+ * @param {String} facilityId UKEF facility ID
+ * @returns {Object} Response object
+ */
 const checkFacilityId = (facilityId) => getAPI(`facility/${facilityId}`);
```

Comment:
The new TFS URLs for this are `deals/${dealId}` and `facilities/${facilityId}` (see my comment above in this file as well)

Reply:
Thanks, happy to resolve this